recipes-kernel/linux/files/patches-5.10/0120-arm64-dts-ti-iot2050-add-the-support-for-the-m.2-var.patch

```diff
@@ -0,0 +1,283 @@
+From 31cfc26cd828660dfe9e1cd92734180a1b4b4973 Mon Sep 17 00:00:00 2001
+From: chao zeng <chao.zeng@siemens.com>
+Date: Tue, 11 Jan 2022 11:28:24 +0800
+Subject: [PATCH 120/120] arm64: dts :ti: iot2050: add the support for the m.2
+ variant
+
+Configure the m2 board dts based on the pg2 advanced variant
+there are three m2 combinations on the iot2050, totally create
+three dts for different combination
```

Comment:
You must first update Documentation/devicetree/bindings/arm/ti/k3.yaml - as a separate patch as it takes a special round in upstream.

Reply:
I have checked the file `k3.ymal`
the info created by TI, should we update this file?

Reply:
I meant something like https://lore.kernel.org/all/5d99e69ff1e2fb78f51f03c351eff1fe1f6c3a71.1632657917.git.jan.kiszka@web.de/

Reply:
OK, picked some patches from upstream and add the info into the yaml file.